app/src/main/java/org/schabi/newpipe/util/Constants.java

```diff
@@ -12,4 +12,6 @@
     public static final String KEY_MAIN_PAGE_CHANGE = "key_main_page_change";
 
     public static final int NO_SERVICE_ID = -1;
+
+    public static final int SECONDS_MIN_LEFT = 10;
```

Comment:
I am not sure if this const should be here. Nevertheless, please add a JDoc comment to it so we can remember what it is used for.
I couldn't find a better name for it, but I think we should write `MINIMUM` instead of `MIN`. Otherwise the MIN is quite confusing in combination with SECONDS :D